Applications/DataExplorer/mainwindow.h

```diff
@@ -1,9 +1,4 @@
 /**
- * \file
```

Comment:
Please keep the `\file` line, otherwise the file comments in the doxygen will also belong to other files.